.github/workflows/benchmark_trigger.yml

```diff
@@ -0,0 +1,112 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Benchmark Trigger
+
+# The privileged workflow listening the PR labeling events and checking if a
+# benchmark label is added. If so, it cancels the running CI workflow on the PR
+# and rerequests it, then the CI workflow will pick up the new benchmark labels.
+# See https://github.com/openxla/iree/issues/10042#issuecomment-1449250094 for
+# the reasons why it is designed in this way.
```

Comment:
rephrase to
1. What the action does
2. How the action is implemented

```suggestion
# Trigger benchmark CI workflows when benchmark PR labels are added.
# When a benchmark label is added, this cancels and then re-runs the CI
# workflow. When the workflow is re-run, it will pick up the latest PR
# description and labels. See
# https://github.com/openxla/iree/issues/10042#issuecomment-1449250094 for
# more background.
```

Also, what is "privileged" about this workflow, compared to other workflows?

Reply:
Thanks for the rephrasing. "Privileged" simply means it is running on the main branch and can have write accesses Github token. Added a line to explain where the workflow will be run.